modules/cmd.js

```diff
@@ -117,12 +164,9 @@ const promise = (
     )
   }
 
-  return Object.freeze({
-    [isCmdSymbol]: true,
-    type: cmdTypes.PROMISE,
-    promiseFactory,
+  return run(promiseFactory, {
```

Comment:
implemented promise, call, arbitrary, and constant using run and action until they are removed.